fmcg_store/demo/sale_order.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_1" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_2" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_3" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_4" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_9" model="sale.order">
+        <field name="partner_id" ref="res_partner_10"/>
+        <field name="pricelist_id" ref="product_pricelist_5"/>
+    </record>
+    <record id="sale_order_10" model="sale.order">
+        <field name="partner_id" ref="res_partner_8"/>
+        <field name="pricelist_id" ref="product_pricelist_5"/>
+    </record>
+    <record id="sale_order_11" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_12" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
+    <record id="sale_order_13" model="sale.order">
+        <field name="partner_id" ref="res_partner_10"/>
+    </record>
+    <record id="sale_order_14" model="sale.order">
+        <field name="partner_id" ref="res_partner_10"/>
+    </record>
+    <record id="sale_order_15" model="sale.order">
+        <field name="partner_id" ref="res_partner_9"/>
+    </record>
```

Comment:
Please keep the sequence while creating multiple records
i.e. sale_order_4, sale_order_5, sale_order_6 and so on..